### PR TITLE
Add support for sphinx and readthedocs

### PR DESCRIPTION
**Pull request by bspidel**
Sphinx is configured and ready for document generation

Issue unfoldingword-dev/door43.org#408

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+# The short X.Y version.
+version = u'1.0.0'
+# The full version, including alpha/beta/rc tags.
+release = u'1.0.0'
+
```
Comment (jag3773) [inline docs/conf.py]: Comment out the above two

File: docs/conf.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+source_suffix = '.rst'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = u'tx-manager'
+copyright = u'2017, Rich Mahn'
+author = u'Rich Mahn'
+
```
Comment (jag3773) [inline docs/conf.py]: Change Rich to unfoldingWord

File: docs/conf.py [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+    #
+    # 'figure_align': 'htbp',
+}
+
+# Grouping the document tree into LaTeX files. List of tuples
+# (source start file, target name, title,
+#  author, documentclass [howto, manual, or own class]).
+latex_documents = [
+    (master_doc, 'tx-manager.tex', u'tx-manager Documentation',
+     u'Rich Mahn', 'manual'),
```
Comment (jag3773) [inline docs/conf.py]: Change Rich to unfoldingWord

**Comment by jag3773** [comment]
Add the whole section on HTML output options from https://github.com/unfoldingWord-dev/Resource-Container/blob/master/docs/conf.py#L117 into conf.py

File: docs/conf.py [hunk 4]
```diff
@@ -0,0 +1,268 @@
...
+# The name of an image file (relative to this directory) to use as a favicon of
+# the docs.  This file should be a Windows icon file (.ico) being 16x16 or 32x32
+# pixels large.
+#
+# html_favicon = None
+
+# Add any paths that contain custom static files (such as style sheets) here,
+# relative to this directory. They are copied after the builtin static files,
+# so a file named "default.css" will overwrite the builtin "default.css".
+html_static_path = ['_static']
```
Comment (phillip-hopper) [inline docs/conf.py:128]: It appears this is causing a warning, may not be important:
```
WARNING: html_static_path entry u'/home/team43/Projects/new-tx-manager/docs/_static' does not exist
```

**Review by phillip-hopper — APPROVED** [review]
LGTM  👍

**Pull request merged**